### PR TITLE
Enhancement: Enable `array_indentation` fixer

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] enables the `array_indentation` fixer
- [x] excludes `manual/en/`
- [x] runs `make coding-standards`

Follows #559.

💁‍♂️ For reference, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.4.0/doc/rules/whitespace/array_indentation.rst.

### CODE REVIEW

File: download-docs.php [hunk 1]
```diff
@@ -43,8 +43,8 @@
 $formats = array(
     "Single HTML file" => "html.gz",
     "Many HTML files"  => "tar.gz",
-#   "Many PDF files"   => "pdf.tar.gz",
-#   "PDF"              => "pdf",
+    #   "Many PDF files"   => "pdf.tar.gz",
+    #   "PDF"              => "pdf",
```
Comment (localheinz) [inline download-docs.php:47]: Perhaps we should remove this instead?

Comment (cmb69) [reply download-docs.php]: Already happened thanks to your PR!

**Review by cmb69 — COMMENTED** [review]
Thank you! Makes perfect sense to me, but I think we shouldn't fix any files under manual/en, since these are just example files taken from the documentation (we may consider to update these from the docs, but that's another story).

**Comment by localheinz** [comment]
@cmb69 

Updated!

**Comment by cmb69** [comment]
Thank you!

**Pull request merged**